include/libopencm3/stm32/dbgmcu.h

```diff
@@ -37,6 +37,23 @@
 #define DBGMCU_IDCODE_DEV_ID_MASK	0x00000fff
 #define DBGMCU_IDCODE_REV_ID_MASK	0xffff0000
 
+/** Known MCUs */
+enum dbgmcu_idcode_dev_id {
+	DBGMCU_IDCODE_DEV_ID_STM32H742 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H743 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H745 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H747 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H750 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H753 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H755 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H757 = 0x450,
+	DBGMCU_IDCODE_DEV_ID_STM32H7A3 = 0x480,
+	DBGMCU_IDCODE_DEV_ID_STM32H7B3 = 0x480,
+	DBGMCU_IDCODE_DEV_ID_STM32H7B0 = 0x480,
+	DBGMCU_IDCODE_DEV_ID_STM32H72X = 0x483,
+	DBGMCU_IDCODE_DEV_ID_STM32H73X = 0x483,
+};
```

Comment:
I don't really feel this is appropriate honestly, at least, certainly not dropped into the h7 pwr commit, and adding a limited subset of h7s to the shared file. I'm not _really_ sure it should ever be defined honestly, but definitely not here in this commit

Reply:
Would you be okay with #defines inside of pwr.c that hold the two constants I care about...something like

```c++
#define DBGMCU_IDCODE_DEV_ID_STM32H74X_5X		0x450
#define DBGMCU_IDCODE_DEV_ID_STM32H7A3_B3_B0		0x480
```

Reply:
yep, was going to say that now that I'd gotten to the bottom, just not fast enough :)